translations/nl-be.yaml

```
@@ -89,12 +93,27 @@ road-sign-concept:
     related-traffic-light: Gerelateerde verkeerslichten
     valid: Gevalideerd
     shape: Vorm
-    dimensions: Dimensies
+    dimensions: Afmetingen
+    height: Hoogte
+    width: Breedte
+    radius: Straal
+    side: Lengte (zijde)
+    unit: Eenheid
+  atLeastOneShapeRequired: You must add at least one shape
+  newShape: New Shape
+  addDimension: Add dimension
+  kind: Kind
+  value: Value
+  saveDimension: Save dimension
+  saveShape: Save shape
+  dimension: Dimension
```

These still need to be translated.

Something like this, maybe?

```suggestion
  atLeastOneShapeRequired: Voeg minstens één vorm toe
  newShape: Nieuwe vorm
  addDimension: Afmeting toevoegen
  kind: Type
  value: Waarde
  saveDimension: Afmeting opslaan
  saveShape: Vorm opslaan
  dimension: Afmeting
```